packages/x-charts/src/LineChart/AreaPlot.tsx

```diff
@@ -18,7 +18,74 @@ export interface AreaPlotSlotProps {
 
 export interface AreaPlotProps
   extends React.SVGAttributes<SVGSVGElement>,
-    Pick<AreaElementProps, 'slots' | 'slotProps'> {}
+    Pick<AreaElementProps, 'slots' | 'slotProps' | 'skipAnimation'> {}
+
+const useCompletedData = () => {
```

Comment:
That's an extraction of the data computation to keep props propagation as clean as possible.

The initial reason was a debugging session, where I suspected to have a forgotten props somwhere